lib/books/author.rb

```diff
@@ -0,0 +1,10 @@
+class Author
+  attr_accessor :first_name, :last_name, :items
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = items
```

Comment:
- [ ] Kindly create these items as an empty array `[]`. It's the array that will store the instances of the Item class.